### DISCUSSION TITLE
Alternative to GPU Temp Limit for 2022 G14

### DISCUSSION THREAD

**Discussion by inasense** [seerge/g-helper#1545] [Q&A]
Hi All, 

Anyone found an effective complement to g-helper to allow this feature on the AMD 6xxx series GPUs besides? My particular chip is in bad need of an LM respread or entire thermal solution replacement, but I'm traveling at the moment and need a temporary solution to keep the GPU from hitting 100c under any major load and crashing the system. Any suggestions/insights much appreciated. Stay safe.

**Comment by seerge** [comment]
@inasense alternative would be limiting GPU power :) (i.e. Platform power slider that is a sum of CPU + GPU)

**Reply by inasense** [reply]
Sure, I guess I'm looking for something to specifically manipulate the GPU without touching the CPU for both stability and diagnostic purposes (trying to sort out of this GPU is failing and an RMA is necessary before warranty runs out). Anyways, this tool you've developed is great, and I appreciate the response.
